user/papers/template.py

```diff
@@ -18,6 +18,7 @@
     "libxml2-progs",
     "meson",
     "pkgconf",
+    "rustfmt",
```

Comment:
just patch this out by removing `cargo_fmt = find_program('cargo-fmt', required: true)`